src/jdbc/cubrid/jdbc/driver/CUBRIDResultSet.java

```diff
@@ -257,6 +257,25 @@ private boolean AfterNext() throws SQLException {
             return false;
         }
 
+        int errorCode = error.getErrorCode();
+        if (errorCode < 0) {
```

Comment:
```suggestion
        if (errorCode != UErrorCode.ER_NO_ERROR) {
```
상수 변수명을 넣어서 작성하면 조금 더 코드가 명확하게 파악될 것 같습니다.

Reply:
네 수정 하도록 하겠습니다.